application/laravel/.env

```diff
@@ -0,0 +1,49 @@
+APP_NAME=Laravel
+APP_ENV=local
+APP_KEY=base64:UDIfX0wCCk+YLsE+/lL6cCmiC5nCpK5MmYP2YLoT/0k=
+APP_DEBUG=true
+APP_URL=http://localhost
+
+LOG_CHANNEL=stack
+
+DB_CONNECTION=mysql
+DB_HOST=127.0.0.1
+DB_PORT=3306
+DB_DATABASE=laravel
+DB_USERNAME=root
+DB_PASSWORD=
+
+BROADCAST_DRIVER=log
+CACHE_DRIVER=file
+QUEUE_CONNECTION=sync
+SESSION_DRIVER=file
+SESSION_LIFETIME=120
+
+REDIS_HOST=127.0.0.1
+REDIS_PASSWORD=null
+REDIS_PORT=6379
+
+MAIL_MAILER=smtp
+MAIL_HOST=smtp.mailtrap.io
+MAIL_PORT=2525
+MAIL_USERNAME=null
+MAIL_PASSWORD=null
+MAIL_ENCRYPTION=null
+MAIL_FROM_ADDRESS=null
+MAIL_FROM_NAME="${APP_NAME}"
+
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+AWS_DEFAULT_REGION=us-east-1
+AWS_BUCKET=
+
+PUSHER_APP_ID=
+PUSHER_APP_KEY=
+PUSHER_APP_SECRET=
+PUSHER_APP_CLUSTER=mt1
+
+MIX_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
+MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
```

Comment:
Are all these environment variables required by laravel even if we don't use them?

Reply:
This is the file that gets generated when a new project is created. I only added the environment variables that I am using in the example. (`DEMO_USERNAME` and `DEMO_PASWORD` i.e.)

Some of these env vars, however, are used, but I am not exactly sure where. Deleting all of them causes a runtime error. So I decided that it's best to alter this file as little as possible.

Reply:
I used an alternative method, which makes changing this file unnecessary.